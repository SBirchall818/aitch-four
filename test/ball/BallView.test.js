import React from 'react';
import renderer from 'react-test-renderer';
import { StyleSheetTestUtils } from 'aphrodite';

import BallView from '../../src/ball/BallView';

describe('BallView :: ', () => {
  beforeEach(() => {
    StyleSheetTestUtils.suppressStyleInjection();
  });

  afterEach(() => {
    StyleSheetTestUtils.clearBufferAndResumeStyleInjection();
  });

  it('should exist', () => {
    expect(BallView).toBeDefined();
  });

  it('should render BallView with defaults', () => {
    const component = renderer.create(<BallView />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
