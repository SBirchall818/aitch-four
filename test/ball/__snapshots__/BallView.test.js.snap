// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BallView ::  should render BallView with defaults 1`] = `
<span
  className="root_936sk"
  style={
    Object {
      "left": 0,
      "top": 0,
    }
  }
/>
`;
